### PR TITLE
NXP-29813 align back on 11.5.x

### PR DESCRIPTION
**Pull request by atchertchian**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <nuxeo.target.version>11.*</nuxeo.target.version>
+    <nuxeo.target.version>[11.3,11.6)</nuxeo.target.version>
```
Comment (ataillefer) [inline pom.xml:17]: why specifying an upper limit?

File: ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java [hunk 2]
```diff
@@ -189,7 +187,9 @@ public void testLivePartialDistribExportAndImport() {
 
     @Test
     public void testLivePartialRefDistribExport() {
-        assumeTrue(isAdminTest());
+        if (!isAdminTest()) {
+            return;
+        }
```
[the remark this answers is not in the document]
Comment (atchertchian) [reply ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java:192]: The test is properly skipped but there are still stack outputs, maybe this is an unwanted side effect of test logging features (?)

For instance on https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/nuxeo-explorer/job/master/203/console
```
20:42:14  2020-12-01 19:42:11,294 [main] ERROR [LogTestWatchman] Test 'org.nuxeo.functionaltests.explorer.nomode.ITExplorerAdminTest#testLivePartialRefDistribExport' failed
20:42:14  org.junit.AssumptionViolatedException: got: <false>, expected: is <true>
20:42:14  	at org.junit.Assume.assumeThat(Assume.java:106) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.Assume.assumeTrue(Assume.java:50) ~[junit-4.13.jar:4.13]
20:42:14  	at org.nuxeo.functionaltests.explorer.nomode.ITExplorerAdminTest.testLivePartialRefDistribExport(ITExplorerAdminTest.java:192) ~[test-classes/:?]
20:42:14  	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
20:42:14  	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
20:42:14  	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
20:42:14  	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
20:42:14  	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26) ~[junit-4.13.jar:4.13]
20:42:14  	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27) ~[junit-4.13.jar:4.13]
20:42:14  	at org.nuxeo.functionaltests.LogTestWatchman$1.evaluate(LogTestWatchman.java:110) [nuxeo-functional-tests-11.3.56.jar:?]
20:42:14  	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306) [junit-4.13.jar:4.13]
20:42:14  	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100) [junit-4.13.jar:4.13]
20:42:14  	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366) [junit-4.13.jar:4.13]
20:42:14  	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103) [junit-4.13.jar:4.13]
20:42:14  	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63) [junit-4.13.jar:4.13]
```

File: ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/BundleGroupArtifactPage.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ public void checkReference(boolean partial, boolean includeReferences, boolean l
                     "Bundle Subgroups\n" + "Bundles");
             checkDocumentationText(null);
             checkSubGroup("org.nuxeo.ecm.platform.comment");
-            checkSubGroup("org.nuxeo.ecm.platform.filemanager");
+            checkSubGroup("org.nuxeo.ecm.directory");
```
[the remark this answers is not in the document]
Comment (atchertchian) [reply ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/BundleGroupArtifactPage.java:65]: Well it is linked actually: these tests rely on the RuntimeFeature and co, and the merging of modules done on 11.x has changed the bundle groups retrieved for some of the bundles used in these tests.

File: pom.xml (hunk 1 above)
Comment (atchertchian) [reply pom.xml:17]: I'm not sure what strategy should be chosen here actually: I'm planning on making some changes to the runtime, that i could use on explorer side, during the 11.5.x lifecycle, and at this point, the package will not be compatible with 11.5.x anymore so I'd rather not have the current version match 11.6 for instance.

That being said, most recent explorer releases match "server-11.*" whereas they're only compatible with >= 11.3, and may not be compatible with 11.6.

Note i would have loved if using `[11.3,11.5)` made it compatible with 11.5.2 for instance, but that's not the case.

File: ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/BundleGroupArtifactPage.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (atchertchian) [reply ftests/explorer-ftests-base/src/main/java/org/nuxeo/functionaltests/explorer/pages/artifacts/BundleGroupArtifactPage.java:65]: Sorry i see what you mean looking again at commits: i'll move this change :+1:

File: ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ataillefer) [reply ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java:192]: Probably due to the use of `LogTestWatchman` yes.

**Review by ataillefer — DISMISSED** [review]

**Review by ataillefer — DISMISSED** [review]

File: ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (atchertchian) [reply ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerAdminTest.java:192]: https://jira.nuxeo.com/browse/NXP-29923

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**